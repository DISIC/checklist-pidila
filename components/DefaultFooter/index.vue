<template src="../../data/layout/footer.html"/>

<script>
  export default {
    name: 'DefaultFooter'
  }
</script>

<style lang="css" scoped>
  @import "../../data/layout/layout-override.css";
</style>
