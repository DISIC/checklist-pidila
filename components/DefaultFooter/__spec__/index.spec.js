import {shallowMount} from '@vue/test-utils'

import DefaultFooter from '../index.vue'

describe('components | DefaultFooter', () => {

  it('should match snapshot', () => {
    // when
    const wrapper = shallowMount(DefaultFooter)

    // then
    expect(wrapper.element).toMatchSnapshot()
  })

})
