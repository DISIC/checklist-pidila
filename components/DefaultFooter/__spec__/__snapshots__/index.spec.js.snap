// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components | DefaultFooter should match snapshot 1`] = `
<footer
  class="footer"
  role="contentinfo"
>
  <div
    class="pidila-footer"
  >
    <div
      class="container"
    >
      <div
        class="footer-content"
      >
        <p>
          La Checklist Pidila est élaborée et maintenue par le
          <a
            href="https://pidila.gitlab.io/"
            title="Pôle intégration de la DILA"
          >
            PiDILA
          </a>
           avec la collaboration de la
          <abbr
            title="Direction interministérielle du numérique"
          >
            DINUM
          </abbr>
           et du
          <abbr
            title="Service d'information du gouvernement"
          >
            SIG
          </abbr>
          . Elle est mise à jour deux fois par an. Contact : pidila@dila.gouv.fr
        </p>

        <p>
          Le
          <a
            href="https://gitlab.com/pidila/checklist-pidila"
          >
            code source
          </a>
           de l'application servant à afficher cette page est distribué sous double licence
          <a
            href="https://gitlab.com/pidila/scampi-twig/blob/master/LICENCE-MIT.md"
          >
            MIT
          </a>
           et
          <a
            href="http://www.cecill.info/licences/Licence_CeCILL-B_V1-fr.html"
          >
            CeCILL-B
          </a>
          . Les contenus sont sous licences respectives de chacun des
          <a
            href="https://pidila.gitlab.io/checklist-pidila/referentiels.html"
          >
            référentiels
          </a>
           alimentant la Checklist Pidila.
        </p>
      </div>

      <div
        class="pidila-logo"
      >
        <a
          href="https://pidila.gitlab/io"
        >
          <svg
            aria-hidden="true"
            class="svg-logo"
            focusable="false"
          >
            <use
              xlink:href="svg/illus-sprite.svg#pidila"
            />
          </svg>

          <span
            class="sr-only"
          >
            Site du Pidila
          </span>
        </a>
      </div>
    </div>
  </div>
</footer>
`;
